ambigauss/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .curves import gaussian, lorentzian
-from .spectra import GaussianSpectrum
+from .spectra import GaussianSpectrum, Lorentzian
```

Comment:
bug here--should be `LorentzianSpectrum`

Reply:
Oh wow, I messed up the easiest thing ever, haha.

Reply:
Fixed :+1: